src/main/java/com/ays/assignment/model/dto/request/UserAssignmentSearchRequest.java

```diff
@@ -0,0 +1,21 @@
+package com.ays.assignment.model.dto.request;
+
+import lombok.Builder;
+import lombok.Data;
+
+/**
+ * A DTO class representing the request data for searching assignment with respect to user's location.
+ * <p>
+ * This class provides getters and setters for the latitude, and longitude fields.
+ * It also includes a builder pattern implementation for constructing instances of this class with optional parameters.
+ * <p>
+ * The purpose of this class is to encapsulate the request data related to searching assignment with respect to user's location, allowing for easy
+ * transfer of the data between different layers of the application.
+ */
+@Data
```

Comment:
Requestlerde `@Data` anotasyonuyla beraber gelen diğer anotasyonlara ihtiyacımız yok ise onları toplu bir şekilde eklemekten kaçınalım. Sadece ihtiyacımız olan anotasyonları ekleyelim. Request için normal şartlar altında `@Getter` ve `@Builder` anotasyonlarının yeterli olması gerekiyor.

Reply:
Gerekli düzeltmeyi yaptım.